nb_anacondacloud/nbextension/uploader.py

```diff
@@ -5,7 +5,7 @@
 try:
     from StringIO import StringIO
 except ImportError:
-    from io import StringIO
+    from io import BytesIO as StringIO
```

Comment:
Importing BytesIO as StringIO could be confusing, so I encourage you to put a little comment here about why you are doing that (and reference the github issue)